packages/trader/src/Modules/Contract/Components/Digits/digits.tsx

```diff
@@ -83,6 +84,7 @@ const DigitsWrapper = ({
                 bid: t.tick,
                 epoch: t.epoch,
                 pip_size: t.tick_display_value?.split('.')[1].length,
+                quote: t.tick,
```

Comment:
```suggestion
                quote: t.quote,
```
?

Reply:
There is not `ask`, `bid` or `quote` in tick (`t`) from `tick_stream`. Instead of these 3 values all we get is field called `tick`. Previously its value was written into `ask` and `bid` (lines 83 and 84), so I just added it into new created `quote` too. This part of code is worked when contract is purchased.
Value for `ask`, `bid` and `quote` is the same in this case and it should be the same and it's expected. We don't want to ask BE for changes, because in other platforms we were using quote too. Plus, it will take a lot of time